src/Gateway/PayplugResponse.php

```diff
@@ -62,6 +62,14 @@ public function process_payment( $resource, $is_payment_with_token = false ) {
 		}
 
 		$metadata = PayplugWoocommerceHelper::extract_transaction_metadata( $resource );
+		$order_metadata = $order->get_meta('_payplug_metadata', true);
+
+		if (is_array($order_metadata) && $order_metadata['transaction_in_progress']) {
+			PayplugGateway::log(sprintf('Order #%s : Order Oney IPN already in progress. Ignoring IPN', $order_id));
```

Comment:
C'est plus lié à Oney ?
```suggestion
			PayplugGateway::log(sprintf('Order #%s : Order IPN already in progress. Ignoring IPN', $order_id));
```

Reply:
Je dirai que non, @ThomasQuadra, tu peux confirmer ?

Reply:
Exact oui, on peut supprimer la notion "Oney" dans le log